main/ser2net/APKBUILD

```diff
@@ -1,35 +1,25 @@
 # Contributor: Carlo Landmeter <clandmeter@gmail.com>
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=ser2net
-pkgver=3.4
-pkgrel=1
+pkgver=3.5.1
+pkgrel=0
 pkgdesc="A proxy that allows telnet/tcp connections to be made to serial ports"
-url="https://sourceforge.net/projects/ser2net"
+url="https://sourceforge.net/projects/$pkgname"
```

Comment:
I prefer to not user `$pkgname` in `url` as I don't think it adds much value. I also often use the `url` to look for new version or find how to report upstream bugs and then i just click on the url. If there is a `$pkgname` there then I will have to copy/paste it to the browser and manually expand it.